zh_cn/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: news_post
+title: "Ruby 1.9.3 的官方支持结束"
+author: "Olivier Lacan"
+translator: "Vincent Zhao"
+date: 2015-02-23 00:00:00 +0000
+lang: zh_cn
+---
+
+从今天开始，对 Ruby 1.9.3 的支持将全部结束。Ruby 新版的 Bug 和安全修复将不会再移植回 1.9.3。
```

Comment:
This change should comment on [L12](https://github.com/ruby/www.ruby-lang.org/pull/1076/files#diff-fa17ad69042c4cb2343a000793c0df77R12) :sweat_smile: 

```
结束维护已经在 => 停止维护已经在
```